pkg/events/event.go

```diff
@@ -88,6 +88,7 @@ const (
 	RebalanceShards                 = "RebalanceShards"
 	DrainNodeRetry                  = "DrainNodeRetry"
 	DrainSubclusterRetry            = "DrainSubclusterRetry"
+	PauseClusterRetry               = "PauseClusterRetry"
```

Comment:
```suggestion
	PauseConnectionsRetry               = "PauseConnectionsRetry"
```
If I am not wrong we are posing connections, right?